test/remote/manifests/e2e-repositories.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argocd-e2e-cluster
+  labels:
+    app.kubernetes.io/name: argocd-e2e-server
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: argocd-e2e-server
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: argocd-e2e-server
+    spec:
+      containers:
+      - name: argocd-e2e-server
+        image: quay.io/jfischer-redhat/argocd-e2e-cluster:latest
```

Comment:
I think we should avoid using personal repo name here.

Reply:
Changed.